tests/unit/trace/tracers/test_context_tracer.py

```diff
@@ -67,6 +67,44 @@ def test_finish_with_spans(self):
 
         self.assertEqual(tracer._spans_list, [])
 
+    def test_finish_with_tracer_subspans(self):
+        tracer = context_tracer.ContextTracer()
+        parent = tracer.start_span('parent')
+        child = tracer.start_span('child')
+        self.assertEqual(child.parent_span, parent)
+        tracer.exporter = mock.Mock()
+        tracer.finish()
+
+        self.assertEqual(tracer.exporter.export.call_count, 2)
+        [[[[c_sd]], _], [[[p_sd]], _]] = tracer.exporter.export.call_args_list
```

Comment:
Subspans created with `Tracer.start_span` get exported separately. This test should pass on all versions.